includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -375,6 +375,7 @@ public function __construct(
 					'large'   => __( 'Large', 'woocommerce-payments' ),
 				],
 			],
+			'platform_checkout_custom_message' => [ 'default' => 'By placing this order, you agree to our [terms_of_service_link] and understand our [privacy_policy_link].' ],
```

Comment:
```suggestion
			'platform_checkout_custom_message' => [ 'default' => __( 'By placing this order, you agree to our [terms_of_service_link] and understand our [privacy_policy_link].', 'woocommerce-payments' ) ],
```
Should we allow for internationalisation/translation for this default message? 🤔

Reply:
Good catch, I forgot about that!